### PR TITLE
Added snap for landscape-client

### PR DESCRIPTION
**Pull request by bceverly**
This PR contains the initial snapification of the client.  It will register but not do much else.

### CODE REVIEW

**Comment by Perfect5th** [comment]
Looks pretty good, but could we get maybe some text in the README about building the snap?

**Comment by bceverly** [comment]
Added details to the README on how to build the snap and push it to the store

File: README [hunk 1]
```diff
@@ -86,3 +86,104 @@ Before opening a PR, make sure to run the full testsuite and lint
 make check3
 make lint
 ```
+
+### Building the Landscape Client snap
+
+First, you need to ensure that you have the appropriate tools installed:
+```
+$ sudo snap install snapcraft
```
Comment (Perfect5th) [inline README:94]: I think we need `--classic`

**Review by Perfect5th — APPROVED** [review]
LGTM

**Pull request merged**